src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -36,10 +36,11 @@
 # (installed by `make prepare`)
 from mock import patch
 
-from toil.common import Config
+from toil.common import Config, Toil
+from toil.job import Job
 from toil.jobStores.abstractJobStore import (AbstractJobStore,
                                              NoSuchJobException,
-                                             NoSuchFileException)
+                                             NoSuchFileException, JobStoreBucketExistsException)
```

Comment:
On further inspection, I learnt that this happened because i used `alt+enter` to import the exception into the test.